build-tools/packages/build-cli/src/commands/merge/branches.ts

```diff
@@ -481,6 +483,8 @@ function getMaybeCiFailuresDescription(props: {
 	return `
 ## ${source}-${target} integrate PR
 
+***DO NOT MERGE THIS PR USING THE GITHUB UI.***
```

Comment:
might be good to note at the end of the instruction how to actually do the merge instead (even if the process is just wait  for someone else to do it)

Reply:
Done.